### ISSUE TITLE
Firewall Logging Default Ingress Deny

### ISSUE THREAD

**Issue by aumohr** [GoogleCloudPlatform/cloud-foundation-fabric#2024]
**Describe the bug**
When a connection is blocked by the default inbound firewall rule, no logging is in place. 

**Rationale**
The default ingress deny firewall rule does not log blocked packets. [...]

The security best practice is to create a low priority catch-all deny firewall rule for ingress [...] with logging enabled. This allows one to audit what traffic is blocked and allows tools like Security Command Center Event Threat Detection to monitor for threats.

**Recommendation**
1) Create a low priority (65534) deny ingress firewall rule with source 0.0.0.0/0 and enable logging, to log blocked incoming connections.

**Expected Behavior**
Create by default a firewall including logging as described under **Recommendation**.

**Comment by juliocc** [comment]
Should we create this through the firewall factory or directly in terraform?

cc @ludoo @sruffilli

**Comment by ludoo** [comment]
Was wondering the same, and I see pros/cons with both approaches.

My vote is for whatever makes the customer's life easier though, so for me factory is a win as it's easier to change and read.
